src/utils/xdrEnumToConstant.js

```diff
@@ -10,6 +11,9 @@ export function xdrEnumToConstant (xdrEnum) {
     xdrEnum.values().forEach(function (item) { res[item.name] = item.value })
     return res
   } catch (error) {
-    throw new Error(`xdrEnumToConstant: Cannot get values from provided xdrEnum (${xdrEnum})`)
```

Comment:
leave the previous version. This place does not localize

Reply:
done